data/src/main/java/com/linkedin/data/template/DataTemplateUtil.java

```diff
@@ -392,6 +415,41 @@ public static TyperefInfo getTyperefInfo(Class<? extends DataTemplate> type)
     return typerefInfo;
   }
 
+  /**
```

Comment:
I'd suggest being a little more clear in this Javadoc: say that this method returns the class of this type's in-memory data representation (perhaps can be rephrased).

Reply:
Fixed